apis/cloud.redhat.com/v1alpha1/clowdapp_types.go

```diff
@@ -250,6 +250,10 @@ type KafkaTopicSpec struct {
 	TopicName string `json:"topicName"`
 }
 
+type TestingSpec struct {
+	IqePlugin string `json:"iqe-plugin"`
```

Comment:
Maybe a nitpick, but I think we're mostly `camelCase` in the specs -- can we go with name `iqePlugin` for the json here?

Reply:
Good catch! Fixed.